drivers/spi/Kconfig.sam

```diff
@@ -19,4 +19,19 @@ config SPI_SAM_DMA
 	help
 	  Enable using DMA with SPI for SPI instances that enable dma channels in
 	  their device tree node.
+
+if SPI_RTIO
+config SPI_SAM_RTIO_SQ_SIZE
```

Comment:
Since this is allocated on the `struct spi_sam_config`, can/should we make this a part of the devicetree binding? Could you see a scenario where we would want different SQ sizes for different buses?

Reply:
I think for this sort of setup we'd really want per instance Kconfig options in some form rather than putting it in DT. The adage from the DT maintainers is that DT is for hardware configuration and description, Kconfig for software options. This is a software option.

So a feature is missing in Kconfig with DT to me to support this, but that doesn't mean we should do it the wrong way I don't think.

Reply:
Ack